endaq/plot/plots.py

```diff
@@ -242,35 +240,43 @@ def get_pure_numpy_2d_pca(df: pd.DataFrame,
     return fig
 
 
-def gen_map(df_map: pd.DataFrame, mapbox_access_token: str, filter_points_by_positive_groud_speed: bool = True,
-            color_by_column: str = "GNSS Speed: Ground Speed") -> go.Figure:
+def gen_map(df_map: pd.DataFrame, mapbox_access_token: str, filter_positive_color_vals: bool = True,
+            lat: str = "Latitude",
+            lon: str = "Longitude",
+            color_by_column: str = "Ground Speed",
+            size_max: float = 15.0,
+            zoom_offset: float = -2.0
+            ) -> go.Figure:
     """
-    Plots GPS data on a map from a single recording, shading the points based some characteristic
+    Plots GPS data on a map from a single recording, shading the points based on some characteristic
```

Comment:
It feels to me like 'characteristic' here reads like an adjective.

```suggestion
    Plots GPS data on a map from a single recording, shading the points based on one of several characteristics
```